src/CustomElements.php

```diff
@@ -1942,7 +1942,7 @@ private function uuidBinToText($value)
 			if (strlen($value) === 16) {
 				return StringUtil::binToUuid($value);
 			}
-			return serialize(array_map('StringUtil::binToUuid', StringUtil::deserialize($value)));
+			return serialize(array_map('\Contao\StringUtil::binToUuid', StringUtil::deserialize($value)));
```

Comment:
Did you test this? Shouldn’t it be without the leading `\`?
```suggestion
			return serialize(array_map('Contao\StringUtil::binToUuid', StringUtil::deserialize($value)));
```

Reply:
In my installation both versions are working.